tests/Fixtures/FileConfigurations/nocollectionoperations.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<resources>
+        <resource class="ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\FileConfigDummy">
+            <itemOperations>
+                <itemOperation name="my_op_name">
+                    <attribute name="method">POST</attribute>
+                </itemOperation>
+            </itemOperations>
+
+            <collectionOperations />
```

Comment:
@sawmurai I think you forgot the `noitemoperations.xml` file in your last commit! 😛